Doc/data/refcounts.dat

```diff
@@ -1193,7 +1192,7 @@ PyList_SET_ITEM:PyObject*:o:0:
 PyList_SetItem:int:::
 PyList_SetItem:PyObject*:list:0:
 PyList_SetItem:Py_ssize_t:index::
-PyList_SetItem:PyObject*:item:0:
+PyList_SetItem:PyObject*:item:$:
 
 PyList_SetSlice:int:::
 PyList_SetSlice:PyObject*:list:0:
```

Comment:
Should we address all the functions that steal references in this PR?

Reply:
Yes, but I'd like some help for that. I don't know all functions that steal references so for now I wanted to just include the syntax and update the comment.

Reply:
Luckily, not too many functions that steal references. You can look through https://docs.python.org/3/search.html?q=steal